testclock/dilated.go

```diff
@@ -219,3 +251,28 @@ func dilateTime(epoch, realNow time.Time,
 	}
 	return epoch.Add(dilatedOffset).Add(time.Duration(float64(delta) / realSecondDuration.Seconds()))
 }
+
+type instant struct {
+	t *time.Time
+	d *time.Duration
+}
+
+func (f instant) Until(now time.Time) time.Duration {
```

Comment:
Comment these - but they just seem to be used internally, so don't need to be exported?